examples/MicroserviceExample/WorkerService/Worker.cs

```diff
@@ -5,7 +5,9 @@
 
 namespace WorkerService;
 
-public partial class Worker : BackgroundService
+#pragma warning disable CA1812 // Avoid uninstantiated internal classes
```

Comment:
As done in other example projects, can you move this to the nowarn section in csproj?

Reply:
There were only one occurrence of the problem, but fixed in eea30be0ca074a6a622024f808006f9a152bfb8f